azure/terraform/storageaccounts.rego

```diff
@@ -23,12 +23,12 @@ azure_issue["storage_secure"] {
 
 storage_secure {
     lower(input.resources[_].type) == "azurerm_storage_account"
-    not azure_attribute_absence["storage_secure"]
```

Comment:
@shahinshirvani this line does not hurt if exist. removing this line also work as default is true.

Reply:
@rezoan 
in this case we need that
assume that ```enable_https_traffic_only``` exists and the value of that is ```false```


```not resource.properties.enable_https_traffic_only``` checks ```enable_https_traffic_only``` didn't exist or the of that is ```false```
so ```azure_attribute_absence["storage_secure"]``` will be true
and ```not azure_issue["storage_secure"]``` will be false

so
```
storage_secure {
    azure_attribute_absence["storage_secure"]
    not azure_issue["storage_secure"]
}
```
has no answer